api/core/number.go

```diff
@@ -94,14 +94,14 @@ func (n Number) AsUint64() uint64 {
 
 // - as x atomic
 
-// AsNumberAtomic gets the raw, uninterpreted raw value. Might be useful for
-// some atomic operations.
+// AsNumberAtomic atomically gets the raw, uninterpreted raw
+// value. Might be useful for some atomic operations.
```

Comment:
maybe rephrase it to `AsNumberAtomic gets uninterpreted raw Number atomically. Might be useful for some atomic operations`

Reply:
Naah, I need to put more "raw" words there. Two times is not enough. ;)

Anyway, thanks for the suggestion. And that sentence about usefulness for atomic ops is a wrong copy pasta, so I dropped it.